src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/controllers/ReportedAdjudicationControllerTest.kt

```diff
@@ -423,6 +423,52 @@ class ReportedAdjudicationControllerTest : TestControllerBase() {
     }
   }
 
+  @Nested
+  inner class CreateHearing {
+    @BeforeEach
+    fun beforeEach() {
+      whenever(
+        reportedAdjudicationService.createHearing(
+          anyLong(),
+          anyLong(),
+          any()
+        )
+      ).thenReturn(REPORTED_ADJUDICATION_DTO)
+    }
+
+    @Test
+    fun `responds with a unauthorised status code`() {
+      createHearingRequest(1, HEARING_REQUEST).andExpect(status().isUnauthorized)
+    }
+
+    @Test
+    @WithMockUser(username = "ITAG_USER", authorities = ["SCOPE_write"])
+    fun `responds with a forbidden status code for non ALO`() {
+      createHearingRequest(1, HEARING_REQUEST).andExpect(status().is5xxServerError) // seems we want to convert 403 to 500?
```

Comment:
For invalid roles, or missing write scope, the convention across HMPPS is to return `403`, so I think we'll want to reinstate `ApiExceptionHandler.handleAccessDenied`